documentation/src/docs/metrics.md

```diff
@@ -254,33 +309,38 @@ public abstract class AbstractStreamMetadataStore implements StreamMetadataStore
                     .thenApply(result -> {
                         if (result.getStatus().equals(CreateStreamResponse.CreateStatus.NEW)) {
                             CREATE_STREAM.reportSuccessValue(1); // Report success event for histogram metric CREATE_STREAM
-                            dynamicLogger.reportGaugeValue(nameFromStream(OPEN_TRANSACTIONS, scope, name), 0); // Report gauge value for Dynamic Gauge metric OPEN_TRANSACTIONS 
+                            dynamicLogger.reportGaugeValue(nameFromStream(OPEN_TRANSACTIONS, scope, name), 0); // Report gauge value for Dynamic Gauge metric OPEN_TRANSACTIONS
                         }
-    
+
                         return result;
                     });
         }
     ...
  }
 ```
 
-### 2.3 Example for Dynamic Meter
-This is an example from io.pravega.segmentstore.server.SegmentStoreMetrics. In this class, we report a Dynamic Meter which represents the segments created.
+## Example for Dynamic Meter
+
+This is an example from `io.pravega.segmentstore.server.SegmentStoreMetrics`. In the class `SegmentStoreMetrics`, we report a Dynamic Meter which represents the Segments created.
+
 ```java
 public final class SegmentStoreMetrics {
     private final DynamicLogger dynamicLogger = MetricsProvider.getDynamicLogger();
-    
+
     public void createSegment() {
             dynamicLogger.recordMeterEvents(this.createSegmentCount, 1);  // Record event for meter metric createSegmentCount
-    } 
+    }
 }
 ```
- 
-# 3. Metric reporter and Configurations
-Reporters are the way through which we export all the measurements being made by the metrics. We currently provide StatsD and CSV output. It is not difficult to add new output formats, such as JMX/SLF4J.
-CSV reporter will export each Metric out into one file. 
-StatsD reporter will export Metrics through UDP/TCP to a StatsD server.
-The reporter could be configured through MetricsConfig.
+
+# Metric Reporter and Configurations
+
+Reporters are used to export all the measurements being made by the metrics. We currently provide [StatsD](https://github.com/b/statsd_spec) and [CSV](https://metrics.dropwizard.io/3.1.0/manual/core/#csv) output. It is not difficult to add new output formats, such as **[JMX](https://metrics.dropwizard.io/3.1.0/manual/core/#jmx)/[SLF4J](https://metrics.dropwizard.io/3.1.0/manual/core/#slf4j)**.
+
+- [**CSV**](https://metrics.dropwizard.io/3.1.0/manual/core/#csv) reporter will export each Metric into one file.
+- [**StatsD**](https://github.com/b/statsd_spec) reporter will export Metrics through UDP/TCP to a StatsD server.
+The reporter could be configured using the `MetricsConfig`.
+
 ```java
 public class MetricsConfig extends ComponentConfig {
```

Comment:
This class was copied in its entirety, and now it doesn't even look like this anymore.

Let's remove this example or at least add a pointer to the actual code file.

Reply:
pointer added